hack/docker-compose.yml

```diff
@@ -108,6 +108,7 @@ services:
       ARCHIVEMATICA_MCPSERVER_SEARCH_ENABLED: "${AM_SEARCH_ENABLED:-true}"
       ARCHIVEMATICA_MCPSERVER_MCPSERVER_PROMETHEUS_BIND_PORT: "7999"
       ARCHIVEMATICA_MCPSERVER_MCPSERVER_PROMETHEUS_BIND_ADDRESS: "0.0.0.0"
+      ARCHIVEMATICA_MCPSERVER_WORKFLOW_FILE: ""
```

Comment:
I don't think we need this line. We don't list environment strings here unless necessary.